azext_edge/edge/providers/orchestration/instances.py

```diff
@@ -71,3 +75,27 @@ def _get_associated_cl(self, instance: dict) -> dict:
         return self.query(
             QUERIES["get_cl_from_instance"].format(resource_id=instance["extendedLocation"]["name"]), first=True
         )
+
+    def update(
+        self,
+        name: str,
+        resource_group_name: str,
+        tags: Optional[dict] = None,
+        description: Optional[str] = None,
+        **kwargs: dict,
+    ) -> dict:
+        instance = self.show(name=name, resource_group_name=resource_group_name)
+
+        if description:
+            instance["properties"]["description"] = description
+
+        if tags:
+            instance["tags"] = tags
+
+        with self.console.status("Working..."):
+            poller = self.iotops_mgmt_client.instance.begin_create_or_update(
+                instance_name=name,
+                resource_group_name=resource_group_name,
+                resource=instance,
+            )
+            return wait_for_terminal_state(poller, **kwargs)
```

Comment:
will we do any error handling like how we did with azure-iot? If I remember correctly, it was for error logging/data purposes.

Reply:
No won't do error handling like that.

Reply:
CLI core has global error handlers that cover azure.core exceptions.